app/views/check_in_out/_check_in_list.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Check In</h1>
```

Comment:
A little late for this app, perhaps, but we find HAML to be much better except in the most complex of cases.